### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-* [98](https://github.com/HBPVIS/zeq/pull/98):
-  Disable zeroconf subscriptions to publishers in the same process
 * [81](https://github.com/HBPVIS/zeq/pull/81):
   Implement (optional) ZeroBuf support
 * [94](https://github.com/HBPVIS/zeq/pull/94):
   Added CellSetBinaryOp to HBP vocabulary
 * [97](https://github.com/HBPVIS/zeq/pull/97):
   Added a small command line tool to send events parsed from a script file.
+* [98](https://github.com/HBPVIS/zeq/pull/98):
+  Disable zeroconf subscriptions to publishers in the same process
```
Comment (eile) [inline doc/Changelog.md]: (sigh)

Changelog should be newest on top. Can you 'list reverse'?

**Comment by tribal-tec** [comment]
ping

**Comment by tribal-tec** [comment]
pingping

**Comment by eile** [comment]
(ooohm)

**Pull request merged**